packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -28,13 +29,21 @@ import {initPeerIdAndEnr} from "./initPeerIdAndEnr.js";
 
 const DEFAULT_RETENTION_SSZ_OBJECTS_HOURS = 15 * 24;
 const HOURS_TO_MS = 3600 * 1000;
+const EIGHT_GB = 8 * 1024 * 1024 * 1024;
 
 /**
  * Runs a beacon node.
  */
 export async function beaconHandler(args: BeaconArgs & GlobalArgs): Promise<void> {
   const {config, options, beaconPaths, network, version, commit, peerId, logger} = await beaconHandlerInit(args);
 
+  const heapSizeLimit = getHeapStatistics().heap_size_limit;
+  if (heapSizeLimit < EIGHT_GB) {
+    logger.warn(
+      "Node.js heap size limit is too low, consider increasing it. See https://chainsafe.github.io/lodestar/faqs#running-a-node"
```

Comment:
Would kinda be nice if we could directly link to the heap limit instead of top level header

Reply:
Not doable with the current docs structure, would require some changes here.